apps/dashboard/src/utils/step.ts

```diff
@@ -1,8 +1,9 @@
-import { ShortIsPrefixEnum, StepResponseDto } from '@novu/shared';
+import { ShortIsPrefixEnum } from '@novu/shared';
 
-const divider = `_${ShortIsPrefixEnum.STEP}`;
+export const WORKFLOW_DIVIDER = `_${ShortIsPrefixEnum.WORKFLOW}`;
+export const STEP_DIVIDER = `_${ShortIsPrefixEnum.STEP}`;
 
-export const getStepBase62Id = (slug: StepResponseDto['slug'] | string = divider) => {
+export const getBase62Id = ({ slug, divider }: { slug: string; divider: string }) => {
```

Comment:
We will soon update the base62 to a shorter version. So let's keep FE agnostic of that change.

```suggestion
export const getEncodedId = ({ slug, divider }: { slug: string; divider: string }) => {
```